opendatahub-io/data-science-pipelines-operator
# Updating v2 manifests to utilize DSPO namePrefix

## The issue resolved by this Pull Request:
Resolves #378 

## Description of your changes:


Updated names of v2 manifests to remove mentions of Kfp, and utlize the `data-science-pipelines-operator-` namePrefix.
Updated `serviceAccountname` to `serviceAccountName` based on an error prompting to do so, while deploying v2.

## Testing instructions

deployed a v2 instance using `make v2deploy` to make sure all resources came up with the `data-science-pipelines-operator-` prefix correctly.

## Checklist
